notebooks/225-distilbert-sequence-classification/225-distilbert-sequence-classification.ipynb

```diff
@@ -0,0 +1,292 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "abc41ac0",
+   "metadata": {},
+   "source": [
+    "# Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fe80a355",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from transformers import AutoTokenizer\n",
+    "import openvino.runtime as ov\n",
+    "import warnings\n",
+    "from pathlib import Path\n",
+    "import numpy as np\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4dde5b43",
+   "metadata": {},
+   "source": [
+    "# Transformers Serialization"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1742fd4b",
+   "metadata": {},
+   "source": [
+    "Check out this link [https://huggingface.co/docs/transformers/serialization] to learn how to convert transformers to onnx format"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5db803ea",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python -m transformers.onnx -h"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4794f066",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "checkpoint = \"distilbert-base-uncased-finetuned-sst-2-english\"\n",
+    "serialize_command = f\"python -m transformers.onnx \\\n",
```

Comment:
notebook is python code, probably we can use this module as function instead as command line tool?